src/react-native/06-building-custom-components/ProdDetails-implementation.tsx

```diff
@@ -0,0 +1,13 @@
+//necessary imports here
+
+const ProdDetails = (props) => {
+  //implementation details here
```

Comment:
These look like TODOs, are they necessary? I think you can skip this comment and add the right imports at the top.